tests/gutenberg_ro.e2e.spec.js

```diff
@@ -0,0 +1,350 @@
+            });
+            it('Author search Results', async function () {
+                if (isJqueryMode) {
+                    console.log('\x1b[33m%s\x1b[0m', '      Test skipped.');
+                    return;
+                }
+                // something is wrong here
+                // search by author name and press enter to apply the filter
+                const filter = await driver.wait(until.elementIsVisible(driver.findElement(By.id('author_filter'))), 1500);
+                await filter.sendKeys(Key.ENTER);
+                const searchList = await driver.wait(until.elementsLocated(By.xpath('//*[@id="books_table"]/tbody')));
+                // revert whatever was typed in the search box and press enter to remove filter
+                await filter.clear();
+                await filter.sendKeys(Key.ENTER);
+                assert.equal(searchList.length, 1);
+            });
+            const downloadFileName = 'Poezii.35323.epub'
+            it('Download EPUB file', async function () {
+                await driver.switchTo().defaultContent();
+                const searchBox = await driver.wait(until.elementIsVisible(driver.findElement(By.id('prefix'))), 1500);
+                await searchBox.clear(1000);
+                await searchBox.sendKeys('Poezii_cover.35323.html');
+                await driver.sleep(1000);
+                // Go and visit the first result of the search
+                const searchList = await driver.wait(until.elementsLocated(By.xpath('//*[@id="articleList"]/a')), 1500);
+                await searchList[0].click();
+                // await searchBox.clear();
+                // if title is not loaded in next 4 seconds then return empty string and fail test
+                await driver.switchTo().frame('articleContent');
+
+                // click on the download button
+                await driver.wait(until.elementLocated(By.xpath('//*[@id="content"]/div/div[2]/div[5]/a[2]')), 1500);
+                const downloadButton = await driver.wait(until.elementIsEnabled(driver.findElement(By.xpath('//*[@id="content"]/div/div[2]/div[5]/a[2]'))), 1500);
+                await downloadButton.click();
+                const downloadFileStatus = await driver.wait(async function () {
+                    // We can only check if the file exist in firefox and chrome (IE and Edge not supported)
+                    // [TODO] only run this part if chrome or firefox
+                    const downloadFileStatus = fs.readdirSync(paths.downloadDir).includes(downloadFileName);
+                    if (downloadFileStatus) fs.rmSync(paths.downloadDir + '/' + downloadFileName);
+                    return downloadFileStatus;
+                }, 5000).catch(() => false);
+                assert.ok(downloadFileStatus);
+
+                // exit if every test and mode is completed
+                if (mode === modes[modes.length - 1]) {
+                    await driver.quit();
```

Comment:
Since you say in the JDDoc header that the function returns, you should add a return here. It is then explicitly returning a Promise to the callier. (If you accept this, please mirror the code to `legacy-ray_charles...`.)

```suggestion
                    return await driver.quit();
```

Reply:
Return is gonna be `void` not `Promise`
IMHO `void` is perfect because we already call the tests from the global scope and we can't use await 

```js
legacyRayCharles.runTests(await loadFirefoxDriver());
gutenbergRo.runTests(await loadFirefoxDriver());
```

Reply:
If you want to change the return type to `Promise<void>` I could do it but then we need to add `await` before every test
- chrome58.bs.runner
- chrome58.bs.runner
- edge18.bs.runner
- etc
One more thing we need to use Top-level await is it ok?